tests/e2e/util.go

```diff
@@ -1355,22 +1355,6 @@ func invokeVCenterServiceControl(command, service, host string) error {
 	return nil
 }
 
-// isFssEnabled invokes the given command to check if vCenter
-// has a particular FSS enabled or not
-func isFssEnabled(host, fss string) bool {
```

Comment:
Keep this code, if needed comment it out.

Reply:
+1